fireant/slicer/queries.py

```diff
@@ -102,12 +104,16 @@ def query_data(self, database, table, joins=None,
             A pd.DataFrame indexed by the provided dimensions paramaters containing columns for each metrics parameter.
         """
         query = self._build_data_query(table, joins or dict(), metrics or dict(), dimensions or dict(),
-                                       dfilters or dict(), mfilters or dict(), references or dict(), rollup or dict())
+                                       dfilters or dict(), mfilters or dict(), references or dict(), rollup or list())
 
         querystring = str(query)
         logger.info("Executing query:\n----START----\n{query}\n-----END-----".format(query=querystring))
 
         dataframe = database.fetch_dataframe(querystring)
+
+        for dimension_key in rollup:
+            dataframe[dimension_key].replace([np.nan], [Totals.key], inplace=True)
```

Comment:
This is tricky because there could be real null values in the query.  I'm not sure off the top of my head how it works with rollup but I think they might get merged together with the totals.  This fixes one problem with null values be labeled as totals when they're not, though.

Could you maybe look into using totals on a dimension with null values in the database?  The currently solution is to expect the user to use Coaesce on the dimension, but would be nice if we could make this automatic, but that isn't ideal.